### DISCUSSION TITLE
v5.52.0

### DISCUSSION THREAD

**Discussion by sokra** [webpack/webpack#14135] [General]
# Feature

* `experiments.executeModule` is enabled by default and the option is removed
  * loaders are now free to use `this.importModule`

# Bugfixes

* fix generated `__WEBPACK_EXTERNAL_MODULE_null__`, which leads to merged externals
* `.webpack[...]` extension is not part of matching and module name

<hr /><em>This discussion was created from the release <a href='https://github.com/webpack/webpack/releases/tag/v5.52.0'>v5.52.0</a>.</em>

**Comment by Jaid** [comment]
Thanks a lot for the `__WEBPACK_EXTERNAL_MODULE_null__` fix. 🙏
I thought I would be alone with this and was about to post an issue for that.
